packages/react/src/examples/forms/validationStatic.stories.tsx

```diff
@@ -172,28 +122,32 @@ export const Static = () => {
           <div className="hds-example-form__grid-6-6">
             <div className="hds-example-form__item">
               <TextInput
+                {...register('firstName')}
+                onChange={(e) => {
+                  handleChange('firstName', e.currentTarget.value);
+                }}
                 id="firstName"
                 name="firstName"
                 label="First name"
-                onChange={formik.handleChange}
-                onBlur={formik.handleBlur}
-                value={formik.values.firstName}
-                invalid={!!getErrorMessage('firstName')}
-                errorText={getErrorMessage('firstName')}
+                invalid={!!errors.firstName?.message}
+                aria-invalid={!!errors.firstName?.message}
+                errorText={errors.firstName?.message}
                 autoComplete="given-name"
                 required
               />
             </div>
             <div className="hds-example-form__item">
               <TextInput
+                {...register('lastName')}
+                onChange={(e) => {
+                  handleChange('firstName', e.currentTarget.value);
```

Comment:
Noticed small mistake here: pointing to `firstName` field 🙂

Reply:
Oh, good spottind :D too much copy & paste it seems :D :D